.github/workflows/terraform-validate-ci.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: terraform-validate-ci
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'terraform/**'
+      - '.github/workflows/terraform-validate-ci.yaml'
+  pull_request:
+    paths:
+      - 'terraform/**'
+      - '.github/workflows/terraform-validate-ci.yaml'
+jobs:
+  job:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: terraform init & validate
+        run: |
+          cd terraform/
+          terraform init -backend=false
```

Comment:
Praise: Thanks for adding `-backend=false`.
Just for my understanding, will this be useful if we ever choose to use something like Google Cloud Storage to maintain Terraform's state, etc?

Reply:
For this `terraform init` step, not that much, in case we want to make sure that this step is working with GCS. Other than that, if we start having an interaction with GCP, it could allow to go step further in our tests with `terraform plan` which requires to be authenticated to GCP. Does that answer your question?

Reply:
Ahh, that makes sense. Thanks, Mathieu!